core/tests/protractor_utils/ProfilePage.js

```diff
@@ -101,6 +113,9 @@ var ProfilePage = function() {
   };
```

Comment:
Can you add a waitFor in `expectToHaveExplorationCardByName` before we call `getText()`?